uvicorn/config.py

```diff
@@ -107,8 +109,14 @@
 
 
 def create_ssl_context(
-    certfile, keyfile, password, ssl_version, cert_reqs, ca_certs, ciphers
-):
+    certfile: Union[Path, str],
```

Comment:
why using different types here and below than those provided in the typeshed ?
```suggestion
    certfile: StrPath,
```

Reply:
`StrPath` is in `_typeshed`, and is not available at runtime by default, so attempting to annotate with `StrPath` may raise `error: Name 'StrPath' is not defined [name-defined]`.

I'm happy to consider it, though. How would I import and use `StrPath`?

Reply:
something like this ?
```
def create_ssl_context(
    certfile: Union[str, os.PathLike[str]],
    keyfile: Optional[Union[str, os.PathLike[str]]],
    password: Optional[str],
    ssl_version: int,
    cert_reqs: int,
    ca_certs: Optional[Union[str, os.PathLike[str]]],
    ciphers: Optional[str],
```

Reply:
Sure, that works -> a2d9085. It's not subscriptable, so just has to be `os.PathLike` instead of `os.PathLike[str]`.